example/gardener-local/gardenlet/values.yaml

```diff
@@ -49,6 +49,9 @@ config:
     # If disabled, a Shoot cluster in local setup won't be able to use the registry cache
     # (it will be still able to pull images but registry cache won't be used).
     ContainerdRegistryHostsDir: true
+  etcdConfig:
+    featureGates:
+      UseEtcdWrapper: true
```

Comment:
Remove this since default behavior should be adhered to, ie, this feature gate will be set to `false`.